src/main/java/org/takes/facets/auth/PsBasic.java

```diff
@@ -112,6 +113,15 @@ public Response exit(final Response response, final Identity identity)
     }
 
     /**
+     * Generate the identity urn.
+     * @param user User
+     * @return URN
+     */
+    private static String generateIdentityUrn(final String user) {
+        return String.format("urn:basic:%s", user);
```

Comment:
@carlosmiranda ok, but I take this idea form `PsBasicTest`. anyway, I will change it